src/main/java/com/timeplus/kafkaconnect/TimeplusSinkTask.java

```diff
@@ -133,20 +141,16 @@ private String getCreateRawPayload(String stream) {
         JSONObject payload = new JSONObject();
```

Comment:
nit: I could not put comment on line 140, so I put it here. Would it make more sense to namet this method `getRawIngestPayload`?